pkg/cmd/roachprod/main.go

```diff
@@ -1171,6 +1172,45 @@ func validateAndConfigure(cmd *cobra.Command, args []string) {
 	}
 }
 
+var updateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "check TeamCity for a new roachprod binary and update if available",
```

Comment:
This (and the `Long` description) could be made a little less implementation focused, which would probably make it more understandable for the main users of this update feature: non-engineering folks that rely on roachprod for experimentation.